recipe-server/client/control/utils/apiFetch.js

```diff
@@ -5,7 +5,9 @@ export default async function apiFetch(url, options = {}) {
 
   const headers = new Headers();
   headers.append('Accept', 'application/json');
-  headers.append('Content-Type', 'application/json');
+  if (!(options.body && options.body instanceof FormData)) {
```

Comment:
We already have `data` vs `body` for specifying different types of data in the request body, why not a `formData` one as well? Alternatively, we should be detecting the type of a single argument and making callers who pass a JSON string explicitly set their content type.

A large refactor of `apiFetch` is out of scope for this PR, though.